### PR TITLE
[DCOS-38720] Add docs for HDFS 2.3.0-2.6.0-cdh5.11.0

### PR DESCRIPTION
**Pull request by elezar**
## Description

https://jira.mesosphere.com/browse/DCOS-38720

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: layout.pug
+navigationTitle: HDFS 2.2.1-2.6.0-cdh5.11.0
```
Comment (mpereira) [inline pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md]: Version is incorrect here.

File: pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+excerpt:
+title: HDFS 2.2.1-2.6.0-cdh5.11.0
```
Comment (mpereira) [inline pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md]: Same.

File: pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md (hunk 1 above)
Comment (elezar) [reply pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md]: 6535313

File: pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md (hunk 2 above)
Comment (elezar) [reply pages/services/hdfs/2.3.0-2.6.0-cdh5.11.0/index.md]: 6535313

**Comment by elezar** [comment]
Thanks @mpereira. See 6535313

**Pull request merged**